### ISSUE TITLE
Alarm on Raspis GPIO...

### ISSUE THREAD

**Issue by ludgerh** [ludgerh/cam-ai#24]
...would easily allow to switch external things, link sound and vision.

**Comment by ludgerh** [comment]
Also, we should prevent the python alarm module to be overwritten during upgrade.

**Comment by sebastianjme** [comment]
Can we send the alarm "in the beginning" of an event. At the moment there is a delay (~ 3 min.)? I think it's because the 60s buffer before the next event. For alarm purposes, it would be nicer to have it more direct. Thx!

**Comment by ludgerh** [comment]
Yes, we can. Any incoming image could be checked for relevant context immediately after coming in and then launch an alarm. The 60 seconds are not mandatory. If you put a lower number (like 10 seconds for example) you might get more events, depending on what your camera sees...

**Comment by ludgerh** [comment]
Done with v1.8.2
